packages/axelar-wasm-std/src/msg_id/tx_hash_event_index.rs

```diff
@@ -22,7 +22,7 @@ impl HexTxHashAndEventIndex {
     }
 }
 
-const PATTERN: &str = "^(0x[0-9a-f]{64})-(0|[1-9][0-9]*)$";
+pub const PATTERN: &str = "^(0x[0-9a-f]{64})-(0|[1-9][0-9]*)$";
```

Comment:
why is it public now?

Reply:
Same question.

Reply:
oh I was using it in nexus gateway, but I guess I'm not now, so I'll revert this